test/functionality/abstract-interpretation/data-frame/data-frame.ts

```diff
@@ -32,56 +32,74 @@ export const DataFrameTestOverapproximation = {
 	rows:     DomainMatchingType.Overapproximation
 };
 
+interface CriterionTestEntry {
+	criterion:  SingleSlicingCriterion,
+	value:      DataFrameDomain,
+	node:       RSymbol<object>,
+	lineNumber: number
+}
+
 export function assertDataFrameDomain(
 	shell: RShell,
```

Comment:
Technically, the tests which do not require a shell can be solely run using `tree-sitter` as the engine (see `assertSliced`). This would allow them to run in parallel, and without any shell startup time. This is probably best done in another issue.